src/css/garden.css

```diff
@@ -56,5 +61,6 @@
 @import "molecules/feedback";
 @import "molecules/collapse";
 @import "molecules/tooltip";
+@import "molecules/steps-stage";
```

Comment:
in this case, it's better to keep the same structure as in `atoms`.

`molecules/steps/stage`